frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/review-and-create.tsx

```diff
@@ -76,11 +80,15 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({
         <ReviewListTitle text="Configure" />
         {(encryption.clusterWide || encryption.storageClass) && (
           <>
-            <ReviewListBody>
+            <ReviewListTitle text="Configure" />
+            <ReviewListBody noValue={!kms.hasHandled}>
               <p className="ocs-install-wizard__review-encryption">Enable Encryption</p>
-              {encryption.advanced && kms.hasHandled && (
-                <p>Connected to external key management service: {kms.name}</p>
+              {encryption.advanced && (
+                <p className="ocs-install-wizard__review-encryption">
+                  Connect to external key management service: {kms.name}
```

Comment:
The KMS connection is done after clicking create, this should be better
```suggestion
                 External key management service: {kms.name}
```